### PR TITLE
[New] Add redux-search@2.4.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by cy91244**
Pull request for issue: #10697

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bvaughn/redux-search
 * Official website (optional, not the repository): https://bvaughn.github.io/redux-search/
 * NPM package url (optional): https://www.npmjs.com/package/redux-search
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 45
   - Count of stars: 1076
   - Count of forks: 51
 * NPM download stats (optional):
   - Downloads in the last day: 39
   - Downloads in the last week: 894
   - Downloads in the last month: 3720

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/redux-search/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "redux-search",
+  "filename": "umd/redux-search.js",
```
Comment (sufuf3) [inline ajax/libs/redux-search/package.json]: Let's use `umd/redux-search.min.js`
We provide minified file as the file name, the current file is not minified.
Thanks.

Comment (cy91244) [reply ajax/libs/redux-search/package.json]: Done, @sufuf3 please confirm the changes, thank you!

File: ajax/libs/redux-search/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bvaughn/redux-search.git"
+  },
+  "npmName": "redux-search",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(commonjs|es)/**/*"
```
Comment (extend1994) [inline ajax/libs/redux-search/package.json]: @PeterDaveHello Should we use `dist/umd` as basePath in this case? (There are other folders under `dist`) Compared to https://github.com/cdnjs/cdnjs/pull/12234#discussion_r158249600.

Comment (sufuf3) [reply ajax/libs/redux-search/package.json]: @cy91244 As the discussion in https://github.com/cdnjs/cdnjs/pull/12234#discussion_r158249600
Let's update this pattern.

Except this part, everything looks good to me.

Many thanks.

**Comment by PeterDaveHello** [comment]
Any updates?

**Comment by cy91244** [comment]
I have modified the PR as the review suggested, thanks.

File: ajax/libs/redux-search/package.json (hunk 2 above)
Comment (extend1994) [reply ajax/libs/redux-search/package.json]: @PeterDaveHello Could you please take a look at this question https://github.com/cdnjs/cdnjs/pull/12316#discussion_r158577616? Thanks.

File: ajax/libs/redux-search/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "name": "redux-search",
+  "filename": "redux-search.min.js",
+  "version": "2.4.0",
+  "description": "Redux bindings for client-side search",
+  "homepage": "https://bvaughn.github.io/redux-search/",
+  "keywords": [
+    "redux",
+    "search",
+    "filter",
+    "react"
```
Comment (sufuf3) [inline ajax/libs/redux-search/package.json]: Let's add more keywords.


Many thanks

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Pull request merged**